test/database.ts

```diff
@@ -1648,7 +1665,7 @@ describe('Database', () => {
     it('should accept gaxOptions', done => {
       const gaxOptions = {};
       database.request = config => {
-        assert.strictEqual(config.gaxOpts, gaxOptions);
+        assert.deepStrictEqual(config.gaxOpts, gaxOptions);
```

Comment:
This is a test that would be a false positive, unless we put in fake properties or revert to comparing that the input object is strictly equal.